### ISSUE TITLE
Incorrect username or password when pulling proxyv2 image during Istio automatic injection

### ISSUE THREAD

**Issue by infa-wayu** [istio/istio#20749]
We have an AWS EKS instance with Istio installed, and automatic injection configured for a specific namespace. When we try to deploy our containers to Kubernetes, we see the following error at the point when the sidecar is being injected:

**Failed to pull image "docker.io/istio/proxyv2:1.4.2": rpc error: code = Unknown desc = Error response from daemon: Get https://registry-1.docker.io/v2/istio/proxyv2/manifests/1.4.2: unauthorized: incorrect username or password**

At this time, checking Docker System Status indicates that all services are functional at this time.

**Expected behavior**
Container successfully deployed with the Istio sidecars injected.

**Steps to reproduce the bug**

** Version: 1.4.2
** Environment: AWS EKS, Kubernetes v.1.14

**Comment by howardjohn** [comment]
I am fairly sure its not private... seems like a docker or EKS issue, can you pull other images?

**Comment by infa-wayu** [comment]
The weirdest thing just happened. I went ahead and deployed the "bookinfo" application to the same namespace, and all the relevant containers were instantiated just fine, meaning the Istio images used for the sidecar were also pulled just fine.

The earlier containers with the pull issue were also fixed as a side effect of deploying "bookinfo" here.

I can only assume that there was some strange transient issue which has been fixed just now.

I'll wait for a couple of days, and will close out the issue if it is not reproducible then.

Thanks for the tip!
